backend/blueprints/spa_api/errors/errors.py

```diff
@@ -17,11 +17,23 @@ def to_dict(self):
         }
 
 
+class NotLoggedIn(CalculatedError):
+    status_code = 403
```

Comment:
I think this should be a 401 or 404.

Reply:
General internet consensus from what I saw said 403 is good for a not logged in request but I can change to 401